spring-cloud-stream-modules-docs/src/main/asciidoc/sources.adoc

```diff
@@ -145,6 +145,42 @@ $$tmpFileSuffix$$:: $$extension to use when downloading files$$ *($$String$$, de
 $$user$$:: $$the username to use$$ *($$String$$, no default)*
 $$withMarkers$$:: $$if true emits start of file/end of file marker messages before/after the data. Only valid with FileReadingMode 'lines'$$ *($$Boolean$$, no default)*
 
+[[spring-cloud-stream-modules-source-tcp]]
+=== TCP
+The `tcp` source acts as a server and allows a remote party to connect to XD and submit data over a raw tcp socket.
```

Comment:
`XD` is wrong word in this context.
Sorry, I don't have any objections and antisympathy to XD, but we don't talk in this sentence about XD at all.